frontend/components/FilterSearch.js

```diff
@@ -0,0 +1,153 @@
+import { useState, useEffect } from 'react'
+import s from 'styled-components'
+import Select from 'react-select/async'
+import fuzzysort from 'fuzzysort'
+
+import { Icon } from './common'
+import { FilterHeader } from './DropdownFilter'
+import {
+  mediaMaxWidth,
+  MD,
+  ANIMATION_DURATION,
+} from '../constants/measurements'
+import {
+  ALLBIRDS_GRAY,
+  MEDIUM_GRAY,
+  CLUBS_RED,
+  CLUBS_GREY,
+  FOCUS_GRAY,
+  BORDER,
+  WHITE,
+} from '../constants/colors'
+
+const SearchWrapper = s.div`
+  margin-bottom: 30px;
+  max-height: 0;
+  opacity: 0;
+  overflow: visible;
+  transition: all ${ANIMATION_DURATION}ms ease;
+
+  ${({ active }) => active && 'max-height: 150vh; opacity: 1;'}
+
+  ${mediaMaxWidth(MD)} {
+    height: auto;
+    overflow: visible;
+    width: 100%;
+    margin: 0;
+    padding: 8px 1rem;
+    border-bottom: 1px solid ${BORDER};
+    position: fixed;
+    left: 0;
+    z-index: 1000;
+    background: ${WHITE};
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.075);
+  }
+`
+
+const SearchIcon = s.span`
+  cursor: pointer;
+  color: ${MEDIUM_GRAY};
+  opacity: 0.5;
+  margin-right: 6px;
+
+  ${mediaMaxWidth(MD)} {
+    right: 24px;
+  }
+`
```

Comment:
Would be good to put a comment above this just explaining why we style things this way (I know I don't really do that but we should get in the habit of it lool)

Reply:
actually I think since moving to feather from fa that we shouldn't have to style things this way anymore, lemme try some things out

Reply:
simplified styles, lmk if you still think it needs a comment